html/semantics/forms/the-textarea-element/value-defaultValue-textContent.xhtml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please rename this to value-defaultValue-textContext-xhtml.xhtml.
Basename-identical tests will cause troubles in WebKit and Blink.